package.json

```diff
@@ -12,19 +12,20 @@
     "lint:fix": "eslint . --fix"
   },
   "dependencies": {
-    "@headlessui/vue": "^1.7.16",
-    "@heroicons/vue": "^2.0.18",
+    "@headlessui/vue": "^1.7.22",
+    "@heroicons/vue": "^2.1.5",
     "@pinia/nuxt": "^0.5.1",
     "@planship/fetch": "^0.2.3",
+    "@planship/nuxt": "0.1.0",
```

Comment:
I believe that Nitro eliminates the need for these to be under dependencies instead of devDependencies. Seems like a stylistic preference, so whatever is standard in the Nuxt ecosystem I guess.

Reply:
I moved them to devDependencies, no strong opinion, but Nuxt ecosystem appears to be indifferent here, this just doesn't matter with Nitro.